exporter/awsxrayexporter/translator/segment.go

```diff
@@ -202,6 +204,14 @@ func MakeSegment(span pdata.Span, resource pdata.Resource) Segment {
 		segmentType = "subsegment"
 	}
 
+	if (attributeDataType == "annotation") {
+		annotations = xrayAttributes
+	} else {
+		metadata = map[string]map[string]interface{}{
+			"default": xrayAttributes,
```

Comment:
Do we need this `default`, how about copying the K-V pairs in as metadata (with my other suggestion the pairs where keys don't match `IndexedAttributes`)?

Reply:
Yes. X-Ray metadata has a "namespace" ahead of key and values. If it's not provided, `default` is used. See [Java SDK implementation](https://github.com/aws/aws-xray-sdk-java/blob/master/aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/entities/EntityImpl.java#L523-L540).

In practice, most of customers use the default namespace which I'm applying here.

> (with my other suggestion the pairs where keys don't match IndexedAttributes)

Agree with that.

Reply:
Thanks, never noticed that!